io/EptReader.cpp

```diff
@@ -613,53 +626,69 @@ uint64_t EptReader::readLaszip(PointView& dst, const Key& key,
     LasReader reader;
     reader.setOptions(options);
 
-    std::lock_guard<std::mutex> lock(m_mutex);
-    reader.prepare(table);
-    const uint64_t startId(dst.size());
+    {
+        std::lock_guard<std::mutex> lock(m_mutex);
+        reader.prepare(table);
+    }
 
-    uint64_t pointId(0);
-    for (auto& src : reader.execute(table))
+    const PointViewSet views(reader.execute(table));
+    if (views.size() != 1)
     {
-        PointRef pr(*src);
-        for (uint64_t i(0); i < src->size(); ++i)
-        {
-            pr.setPointId(i);
-            process(dst, pr, nodeId, pointId);
-            ++pointId;
-        }
+        throwError("Unexpected view count: " + key.toString());
```

Comment:
This error message doesn't explain what someone did wrong or how to correct.

Reply:
I updated it. It is really a fail safe check, I don't think it would ever happen and I don't know how such a thing could happen.